ansible/roles/runner/files/app_token.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
In my opinion, this kind of code should be written in Python which has more native support for HTTP requests, JSON, etc.

Reply:
This is based of the work I did in https://github.com/myoung34/docker-github-actions-runner/pull/254 , used bash because this is what upstream was using. 
I would rather not recreate a custom version for our use case. What I should probably do, now that it has been merged, is no keep the internal version here, put pull it directly from upstream, e.g from https://raw.githubusercontent.com/myoung34/docker-github-actions-runner/master/app_token.sh